packages/FlexGrid/FlexGrid.modules.scss

```diff
@@ -2,9 +2,6 @@
 
 .flexGrid {
   padding: 0;
-}
-
-.centre {
   margin: 0 auto;
```

Comment:
Why is `margin: 0 auto` on both the `flexGrid` AND the `limitWidth` classes? It only needs to be there once right?